packages/flame_sprite_fusion/example/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Miscellaneous
```

Comment:
Same, only the root gitignore should be needed

Reply:
Removed it.